server/src/main/java/io/druid/server/initialization/jetty/JettyServerModule.java

```diff
@@ -269,7 +269,7 @@ static Server makeAndInitializeServer(
       httpsConfiguration.setRequestHeaderSize(config.getMaxRequestHeaderSize());
       final ServerConnector connector = new ServerConnector(
           server,
-          new SslConnectionFactory(sslContextFactory, HttpVersion.HTTP_1_1.toString()),
+          new SslConnectionFactory(sslContextFactory, HTTP_1_1_STRING),
```

Comment:
part of PR?

Reply:
https://github.com/druid-io/druid/pull/5382#discussion_r168253264